lib/provisioner/worker/plugins/automators/chef_solo_automator/resources/cookbooks/cdap/metadata.json

```diff
@@ -49,7 +49,5 @@
   "recipes": {
 
   },
-  "version": "2.28.4",
-  "source_url": "https://github.com/caskdata/cdap_cookbook",
-  "issues_url": "https://issues.cask.co/browse/COOK/component/10603"
```

Comment:
oh, used an older knife version that must've stripped them on the import (?).  they are there in the actual cookbook release, so minimal impact